db/postgres/offerings.go

```diff
@@ -105,7 +107,7 @@ func (p PostgresDBStore) GetExpandedOfferingsByUserID(id string) ([]*model.Expan
 func (p PostgresDBStore) GetOfferingsByDateRange(start time.Time, end time.Time) ([]*model.Offering, error) {
 	sqlStatement :=
 		`SELECT id, user_id, workspace_id, start_time, end_time, cancelled, created_by FROM offerings 
-				WHERE start_time >= $1 AND end_time <= $2;`
+				WHERE start_time <= $1 AND end_time >= $2;`
```

Comment:
User perspective. Do  we want this to only be offerings that atleast contain the start end time, which sounds reasonable. Or do we want to accomodate everything? I think this makes sense though